src/lib/SX1280Driver/SX1280.cpp

```diff
@@ -392,23 +399,32 @@ void ICACHE_RAM_ATTR SX1280Driver::TXnb()
     if (currOpmode == SX1280_MODE_TX) //catch TX timeout
     {
         //DBGLN("Timeout!");
-        SetMode(SX1280_MODE_FS);
+        SetMode(SX1280_MODE_FS, SX1280_Radio_All);
         TXnbISR();
         return;
     }
+
+    if (GPIO_PIN_NSS_2 != UNDEF_PIN)
+    {
+        if (lastSuccessfulPacketRadio == SX1280_Radio_1) instance->SetMode(SX1280_MODE_FS, SX1280_Radio_2); // Make sure the unused radio is in FS mode and will not receive the tx packet.
+        if (lastSuccessfulPacketRadio == SX1280_Radio_2) instance->SetMode(SX1280_MODE_FS, SX1280_Radio_1);
```

Comment:
Change to `SX1280_Radio_All - SX1280_Radio_2`.  If 3 or 4 radios are added, then all non used can be set to SX1280_MODE_FS in a single SPI transfer.

Reply:
Not possible yet with out reworking WaitOnBusy.